components/nav-list/src/index.scss

```
@@ -50,3 +54,8 @@
   padding-inline-start: var(--utrecht-nav-bar-link-padding-inline-start);
   text-decoration: none;
 }
+
+.utrecht-nav-list__link--current {
+  background-color: var(--utrecht-navigation-link-is-current-background-color, transparent);
+  font-weight: var(--utrecht-navigation-link-is-current-font-weight);
```

`is-current` kan `current` worden, om de conventies te volgen

```suggestion
  background-color: var(--utrecht-navigation-link-current-background-color, transparent);
  font-weight: var(--utrecht-navigation-link-current-font-weight);
```